ss/test/storage_bench_test.go

```diff
@@ -0,0 +1,194 @@
+//go:build rocksdbBackend
+// +build rocksdbBackend
```

Comment:
can we remove the build tag?

Reply:
[]()

Reply:
Its now updated without build tags and not backend specific